src/main/java/com/google/sps/servlets/UserDataServlet.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserServiceFactory;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.MediaType;
+import java.io.IOException;
+
+/**
+ * Servlet that retrieves the email address of the user.
+ */
+@WebServlet("/user")
+public class UserDataServlet extends HttpServlet {
```

Comment:
This is ok for a demo in my opinion but we should also make this a proper REST API with JSON later.

Reply:
I thought about serializing into JSON the [User](https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/users/User.html) object, but probably it contains some sensitive information and we only need the email address.

Reply:
agree. I wouldn't expose more than you absolutely need. But you could serialize it as a JSON like this:

```
{
  email: "something@something.else"
}
```

Reply:
Yes, I did it.

Reply:
(Sorry I somehow clicked the wrong _View Changes_ button and didn't see the last commit. Added a few more comments now.